packages/registry/registry/dot-ui/blocks/address-input/components/address-input.dedot.tsx

```
@@ -0,0 +1,60 @@
+
+// Identity chain for Dedot
+const IDENTITY_CHAIN = "paseo_people" as const;
+
+export function AddressInput(props: AddressInputProps) {
+  const dedotContext = useDedot();
+
+  // Create elegant services object with proper typing
+  const services = useMemo(
+    () => ({
+      useIdentity: (address: string) =>
+        usePolkadotIdentity(address, IDENTITY_CHAIN),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)